databases/postgresql10/Portfile

```diff
@@ -1,21 +1,20 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem 1.0
-PortGroup select 1.0
```

Comment:
If you remove the "select" PortGroup then if you remove the "select" PortGroup you also need to remove the runtime dependency on the "postgresql_select-CY" port and the actual select files in the "files" directory.

Reply:
I wrote this mirroring what the Python group portfiles are doing. They still have the runtime dependency to pull in their stuff without the select group.

Agreed about the select files in files/